### PR TITLE
1266-Iceberg-should-not-use-Spec-internals

### LINKED ISSUE #1266: Iceberg should not use Spec internals
Spec directly use a value holder of Spec instead of using Spec public API. This will break with Spec 2.

### PR DESCRIPTION
**Pull request by jecisc**
Remove call to Spec internals.

Fixes #1266

### CODE REVIEW

**Review by gcotelli — APPROVED** [review]

File: Iceberg-Plugin-Pharo/IcePharoNewBranchFromIssuePanel.class.st [hunk 1]
```diff
@@ -96,7 +96,7 @@ IcePharoNewBranchFromIssuePanel >> initializeWidgetsContents [
 		whenBuiltDo: [ :w | 
 			w widget wrapFlag: false.
 			w widget enabled: false ].
-	issueNumberText textHolder whenChangedDo: [ :text | self validateIssue: text ]
+	issueNumberText whenTextChangedDo: [ :text | self validateIssue: text ]
```
Comment (tinchodias) [inline Iceberg-Plugin-Pharo/IcePharoNewBranchFromIssuePanel.class.st:99]: In dev-1.9, this change is already applied

File: Iceberg-Plugin-GitHub/IceGitHubNewBranchFromIssuePanel.class.st [hunk 2]
```diff
@@ -109,8 +109,7 @@ IceGitHubNewBranchFromIssuePanel >> initializeWidgetsContents [
 		whenBuiltDo: [ :w | 
 			w widget wrapFlag: false.
 			w widget enabled: false ].
-	issueNumberText textHolder
-		whenChangedDo: [ :text | self validateIssue: text ]
+	issueNumberText whenTextChangedDo: [ :text | self validateIssue: text ]
```
Comment (tinchodias) [inline Iceberg-Plugin-GitHub/IceGitHubNewBranchFromIssuePanel.class.st:112]: In dev-1.9, this change is already applied

**Comment by tinchodias** [comment]
Should we close?

**Comment by tinchodias** [comment]
Thanks for the changes, they are in dev-1.9.

**Pull request closed without merging**